MAUI/ListView/item-drag-and-drop.md

```diff
@@ -0,0 +1,485 @@
+    e.Cancel = true;
+}
+{% endhighlight %}
+{% endtabs %}
+
+## Cancel dropping for the dragged item
+
+To cancel dropping for the dragged item, handle the `ItemDragging` event based on the conditions of `Action` event argument. 
+
+You can cancel the dropping action for a particular item by setting the `Cancel` property of the `ItemDraggingEventArgs`.
+
+{% tabs %}
+{% highlight c# %}
+using Syncfusion.Maui.ListView.Helpers;
+private void ListView_ItemDragging(object sender, ItemDraggingEventArgs e)
+{
+  // Cancel the dropping if drop the drag item into out of view.
+  var listView = sender as ListView;
+  var totalExtent = this.listView.GetVisualContainer().Bounds.Bottom;
+  if (e.Action == DragAction.Drop && (e.Bounds.Y < -30 || e.Bounds.Bottom > totalExtent + 40))
+    e.Cancel = true;
+}
+{% endhighlight %}
+{% endtabs %}
+
+## Reorder the underlying collection
+
+The underlying collection can be reordered directly by setting the `UpdateSource` property to `true`. The default value is `false`. 
+
+{% tabs %}
+{% highlight xaml %}
+<ContentPage xmlns:syncfusion="clr-namespace:Syncfusion.Maui.ListView;assembly=Syncfusion.Maui.ListView">
+  <syncfusion:SfListView x:Name="listView" 
+                   ItemsSource="{Binding ToDoList}"
+                   DragStartMode="OnHold"
+                   BackgroundColor="#FFE8E8EC"
+                   ItemSize="60">
+  <syncfusion:SfListView.DragDropController>
+                <syncfusion:DragDropController
+                    UpdateSource="True">
+                </syncfusion:DragDropController>
+   </syncfusion:SfListView.DragDropController>
+</syncfusion:SfListView>                
+</ContentPage>
+{% endhighlight %}
+{% highlight c# %}
+this.listView.DragDropController.UpdateSource = true;
+{% endhighlight %}
+{% endtabs %}
+
+We can able to update collection even when `UpdateSource` is `false`. Like, user can decide where dragged item should be dropped actually by handling the `ItemDragging` event with `DragAction.Drop`.
+
+{% tabs %}
+{% highlight c# %}
+private void ListView_ItemDragging(object sender, ItemDraggingEventArgs e)
+{
+   if (e.Action == DragAction.Drop)
+      {
+        viewModel.ToDoList.MoveTo(1, 5);
+      }
+}
+{% endhighlight %}
+{% endtabs %}
+
+N> Underlying collection will not be updated when any data operation like sorting or grouping is performed. The order will be maintained only in `DisplayItems` of data source. When drag and drop an item between groups, the value of the property in which grouping is performed is updated in data object. 
+
+## Delete item when dropping in particular view
+
+To delete the dragged item when dropping into a particular view, handle the `ItemDragging` event based on the conditions of `Action` and `Bounds` event arguments. 
+
+To delete the dragged item from the underlying collection when dropping into delete icon, follow the code example. It will enable or disable whenever drag started, and dropped by `IsVisible` property in ViewModel.
```

Comment:
To delete the dragged item from the underlying collection when dropping into the delete icon, follow the code example. It will enable or disable whenever drag started, and dropped by the `IsVisible` property in ViewModel.

Reply:
Addressed the concerns provided.